build-logic/src/main/kotlin/com.hedera.hashgraph.conventions.gradle.kts

```diff
@@ -13,120 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import java.text.SimpleDateFormat
 import java.util.Date
 
 plugins {
-    `java-library`
-    jacoco
-    id("com.hedera.hashgraph.repositories")
-    id("com.hedera.hashgraph.spotless-conventions")
-    id("com.hedera.hashgraph.spotless-java-conventions")
-    id("com.hedera.hashgraph.spotless-kotlin-conventions")
-    id("com.hedera.hashgraph.jpms-modules")
-    id("com.hedera.hashgraph.jpms-module-dependencies")
+    id("java-library")
+    id("com.hedera.hashgraph.java")
```

Comment:
Because it is also used as base for `com.hedera.hashgraph.application` which should not be a `java-library`.

The main reason is that `java-library` adds the ability to declare `api` (`requires transitive`) dependencies. For applications you usually do not want/need this. And the _dependency analysis plugin_ should not suggest using them.